pyproject.toml

```
@@ -1,12 +1,12 @@
 [tool.poetry]
 name = "pydanticrud"
-version = "0.4.2"
+version = "0.4.3"
```

```suggestion
version = "1.0.0"
```
Let's communicate the gravity of this change.

---

Done